docs/roadmap.md

```diff
@@ -11,37 +11,49 @@ Here's how we intend to roll out Project Reunion:
 <table>
   <tbody>
     <tr>
-      <th>//BUILD 2020</th>
-      <th>End of 2020</th>
-      <th>2021++</th>
+      <th>Reunion 0.5 (March 2021)</th>
+      <th>Reunion 0.8 (Summer 2021)</th>
+      <th>Reunion 1.0 (Fall 2021)</th>
+      <th>2022 and Beyond</th>
     </tr>
     <tr>
-      <td>
+    <td>
+      <ul>
+            <li>1st supported public release</li>
+            <li>WinUI 3 to work in Desktop Packaged (Win32) Apps</li>
+            <li>Updates to MSIX to easily have their apps updates and repair</li>
+            <li>WinUI 3 and Win2D will be part of Reunion</li>
+       </ul>
+     </td>
+     <td>
         <ul>
-            <li>WinUI preview for UWP & Win32</li>
-            <li>Windows Virtual Desktop</li>
-            <li>WebView2 Preview</li>
-            <li>Project Reunion repo</li>
+            <li>Better bridging the gap between Win32 and WinRT</li>
+            <li>Delivery of Runtime dependencies for MSIX, Win32</li>
+            <li>State Notifications is supported</li>
+            <li>Framework Package Support</li>
+            <li>Enabling developers to install the runtime packages with their app</li> 
          </ul>
      </td>
-      <td>
+     <td>
         <ul>
-            <li>WinUI "go-live" Preview</li>
-            <li>Project Reunion Preview</li>
+            <li>Reunion Windowing release</li>
+            <li>New Rendering and Input features</li>
```

Comment:
Any details on rendering & input?

Reply:
If there's no concrete details yet, I'm fine with leaving this as you wrote it till we obtain those details. We need a quick first update of our roadmap (since it's so outdated) and then we can refine it next week